apps/collection/views/confirm.html

```diff
@@ -1,200 +1,17 @@
-          {{/values.date-of-birth}}
-          {{#values.nationality}}
-          <tr>
-            <td>{{#t}}pages.check-details.table.headers.nationality{{/t}}</td>
-            <td>{{values.nationality}}</td>
-            <td><a href="personal-details/edit#nationality" class="button">{{#t}}buttons.change.nationality{{/t}}</a></td>
-          </tr>
-          {{/values.nationality}}
-          {{#values.use-address}}
-          {{#values.contact-address-street}}
-            <tr>
-              <td>{{#t}}pages.check-details.table.headers.contact-address{{/t}}</td>
-              <td>
-                {{values.contact-address-house-number}} {{values.contact-address-street}},
-                {{values.contact-address-town}},
-                {{#values.contact-address-county}}{{values.contact-address-county}},{{/values.contact-address-county}}
-                {{values.contact-address-postcode}}
-              </td>
-              <td><a href="contact-details/edit#address-group" class="button">{{#t}}buttons.change.address{{/t}}</a></td>
-            </tr>
-          {{/values.contact-address-street}}
-          {{/values.use-address}}
-          {{#values.passport}}
-          <tr>
-            <td>{{#t}}pages.check-details.table.headers.passport{{/t}}</td>
-            <td>{{values.passport}}</td>
-            <td><a href="personal-details/edit#passport" class="button">{{#t}}buttons.change.passport{{/t}}</a></td>
-          </tr>
-          {{/values.passport}}
-          {{#values.email}}
-          <tr>
-            <td>{{#t}}pages.check-details.table.headers.email{{/t}}</td>
-            <td>{{values.email}}</td>
-            <td><a href="contact-details/edit#email" class="button">{{#t}}buttons.change.email{{/t}}</a></td>
-          </tr>
-          {{/values.email}}
-          {{#values.phone}}
-            <tr>
-              <td>{{#t}}pages.check-details.table.headers.phone{{/t}}</td>
-              <td>{{values.phone}}</td>
-              <td><a href="contact-details/edit#phone" class="button">{{#t}}buttons.change.phone{{/t}}</a></td>
-            </tr>
-          {{/values.phone}}
-        </tbody>
-      </table>
-
-      {{<partials-org-type}}{{/partials-org-type}}
-
-      <h2>{{#t}}footer.title{{/t}}</h2>
-      <p>{{#t}}footer.explanation{{/t}}</p>
-
-      {{#input-submit}}send{{/input-submit}}
-
-    {{/form}}
-
-  {{/partials-form}}
-
-
-{{/content}}
-
-{{/layout}}
\ No newline at end of file
+{{<partials-page}}
+  {{$journeyHeader}}
+    {{#t}}journey.collection.header{{/t}}
+  {{/journeyHeader}}
+  {{$page-content}}
+    <p>{{#t}}pages.confirm.subheader{{/t}}</p>
+        {{#rows}}
+          {{> partials-summary-table}}
+        {{/rows}}
+    <h2>{{#t}}fields.org-help.legend{{/t}}</h2>
+    {{#radio-group}}org-help{{/radio-group}}
+    {{> partials-org-type}}
+    <h2>{{#t}}footer.title{{/t}}</h2>
+    <p>{{#t}}footer.explanation{{/t}}</p>
+    {{#input-submit}}send{{/input-submit}}
+  {{/page-content}}
+{{/partials-page}}
```

Comment:
Nice

is this smaller now because of the summary component you're using?

Reply:
Yes, that is correct - previously the the confirm page was pretty much hard coded where now it pulls in the items-table.html partial from hof which pulls in config from the summary commponent